.github/workflows/main.yaml

```diff
@@ -16,24 +16,54 @@ jobs:
         working-directory: ${{ matrix.ubuntu-version }}
     steps:
       - uses: actions/checkout@v2
-      - name: Extract build targets
+      - name: Extract targets
         run: |
+          TAG_MINIMAL=$(cat TAG_MINIMAL)
+          c="$(../tag_exists ubuntu-minimal $TAG_MINIMAL)"
+          if [ "$c" = ng ]; then
+              echo "copy ubuntu-minimal:$TAG_MINIMAL"
+              echo yes! > COPY_MINIMAL
```

Comment:
What do you mean `COPY_MINIMAL`?
I guess `NEED_UPDATE_MINIMAL`.

Reply:
@zoetrope 
Yes. Your guess is correct. I fixed it. 758555c